kafka/pykafka/cluster.py

```diff
@@ -95,6 +98,38 @@ def _update_topics(self, metadata):
             else:
                 self._topics[name].update(meta)
 
+    def get_offset_manager(self, consumer_group_name):
```

Comment:
In broker.py this is `group` and here it's `consumer_group_name`.  There's no ConsumerGroup type, so `_name` is needless specification.  It'd be best to standardize on simply `consumer_group`.